pandas/core/accessor.py

```diff
@@ -162,7 +166,7 @@ class CachedAccessor:
         the single argument ``data``.
     """
 
-    def __init__(self, name, accessor):
+    def __init__(self, name, accessor) -> None:
```

Comment:
can we say anything about name?  str?  Optional[Hashable]?

Reply:
```str``` worked